api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/ContentUpgrade.java

```diff
@@ -215,6 +216,48 @@
      */
     ContentUpgrade doReplaceValuesOfMultiValueProperty(String name, String[] oldValues, String[] newValues);
 
+    /**
+     * Replaces a substring in all properties of the matching resource. Only applies to String
+     * properties.
+     * 
+     * @param oldValue old value
+     * @param newValue new value
+     * @return upgrade object
+     */
```

Comment:
I would rename this with ...AllProperties so that it's clear directly from the method name

Reply:
True, updated method names